### DISCUSSION TITLE
Mobile app

### DISCUSSION THREAD

**Discussion by shhhiiiran** [freenet/freenet-core#420] [Q&A] [answered]
I came across Locutus doing some research regarding censorship-free app development as the newer version of Freenet.

Can Locutus be installed on a mobile device, e.g. Android?

Maybe more of a light version of Locutus (running in dark mode! :D ). Light version, as in, it is tight to the application running in Locutus, not the full node with a total capacity. 
Can a mobile app (using something like Capacitor) be created using Locutus without having a full node running on a local computer or the device itself? So the mobile app would be decentralized, censorship-free, anonymous, and available on the regular Locutus network.

**Comment by AshtonSnapp** [comment]
Something like this would probably be doable on Android. I'm not entirely sure about iOS, considering how locked down Apple makes everything...

**Comment by iduartgomez** [comment, marked as answer]
We hope to make mobile support first class. Whereas a limited node will be running in the device, the network model already kind of is "pay only for what you use" (resource wise) but we will see what kind of optimizations we can bring in the future, node daemons are supposed to be lightweight and they can be potentially embedded/used as a library architecture wise.
